README.md

```diff
@@ -44,6 +44,29 @@ to deploy it to a cluster.
 
 ## Local development
 
+### Option 1: VSCode Remote Container
```

Comment:
I would call this option 2 since it relies on a VSCode plugin (and requires VSCode).